BREAKING.md

```diff
@@ -28,6 +29,9 @@ See [AgentScheduler-related deprecations](#AgentScheduler-related-deprecations)
 ### Changes to client-api
 - The `load` function from `document.ts` will fail the container does not exist. Going forward, please use the `create` function to handle this scenario.
 
+### Property removed from the ContainerRuntime class
+- the `existing` property from `ContainerRuntime` has been removed. Inspecting this property in order to decide whether or not to perform initialization operations should be replaced with extending the `RuntimeFactoryHelper` abstract class from `@fluidframework/runtime-utils` and overriding `instantiateFirstTime` and `instantiateFromExisting`.
```

Comment:
Might want to reference IContainerRuntimeFactory.instantiateRuntime and existing flag there